next.config.js

```
@@ -31,24 +31,12 @@ const execSync = require("child_process").execSync;
 const GIT_COMMIT_SHA = NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA
   ? NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA
   : execSync("git rev-parse HEAD").toString().trim();
-// Can not use (??) syntax because of cypress plugin
-const GIT_BRANCH = NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF
-  ? NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF
-  : execSync("git rev-parse --abbrev-ref HEAD").toString().trim();
-
-// compute the release stage of the app
-const RELEASE_STAGE =
-  NODE_ENV === "production" && GIT_BRANCH === "production"
-    ? "production"
-    : NODE_ENV === "production" && GIT_BRANCH === "staging"
-    ? "staging"
-    : "development";
 
 const isOnStagingOrProduction =
   RELEASE_STAGE === "staging" || RELEASE_STAGE === "production";
 
 const BUILD_ID = encodeURIComponent(
-  `${APP_VERSION}-${GIT_BRANCH}.${GIT_COMMIT_SHA}`
+  `${APP_VERSION}-${RELEASE_STAGE}.${GIT_COMMIT_SHA}`
```

The one thing I'm wondering is if we should fail the build if the RELEASE_STAGE is not set, to protect from accidentally publishing without one?

---

Good idea. I created a quick PR for this. Can you review? https://github.com/NYPL-Simplified/circulation-patron-web/pull/512

---

Hm for some reason making AXISNOW_DECRYPT throw an error caused the docker build step to fail when I merged that PR to `staging`. I'm reverting the change for now https://github.com/NYPL-Simplified/circulation-patron-web/pull/513